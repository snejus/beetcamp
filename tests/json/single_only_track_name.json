{
  "@context": "https://schema.org",
  "@id": "https://gutkeinforu.bandcamp.com/track/oenera",
  "@type": "MusicRecording",
  "additionalProperty": [
    {
      "@type": "PropertyValue",
      "name": "track_id",
      "value": 793629957
    },
    {
      "@type": "PropertyValue",
      "name": "art_id",
      "value": 3989527092
    },
    {
      "@type": "PropertyValue",
      "name": "license_name",
      "value": "all_rights_reserved"
    }
  ],
  "byArtist": {
    "@id": "https://gutkeinforu.bandcamp.com",
    "@type": "MusicGroup",
    "name": "GUTKEIN"
  },
  "copyrightNotice": "All Rights Reserved",
  "dateModified": "04 Mar 2023 01:32:43 GMT",
  "datePublished": "10 Jan 2021 00:00:00 GMT",
  "description": "little distorted",
  "duration": "P00H05M55S",
  "image": "https://f4.bcbits.com/img/a3989527092_10.jpg",
  "inAlbum": {
    "@type": "MusicAlbum",
    "albumRelease": [
      {
        "@id": "https://gutkeinforu.bandcamp.com/track/oenera",
        "@type": [
          "MusicRelease",
          "Product"
        ],
        "additionalProperty": [
          {
            "@type": "PropertyValue",
            "name": "item_id",
            "value": 793629957
          },
          {
            "@type": "PropertyValue",
            "name": "item_type",
            "value": "t"
          },
          {
            "@type": "PropertyValue",
            "name": "selling_band_id",
            "value": 1309392494
          },
          {
            "@type": "PropertyValue",
            "name": "type_name",
            "value": "Digital"
          },
          {
            "@type": "PropertyValue",
            "name": "art_id",
            "value": 3989527092
          }
        ],
        "description": "Includes high-quality download in MP3, FLAC and more. Paying supporters also get unlimited streaming via the free Bandcamp app.",
        "image": [
          "https://f4.bcbits.com/img/a3989527092_10.jpg"
        ],
        "musicReleaseFormat": "DigitalFormat",
        "name": "OENERA",
        "offers": {
          "@type": "Offer",
          "availability": "OnlineOnly",
          "price": 1.0,
          "priceCurrency": "EUR",
          "priceSpecification": {
            "minPrice": 1.0
          },
          "url": "https://gutkeinforu.bandcamp.com/track/oenera#t793629957-buy"
        }
      },
      {
        "@id": "https://gutkeinforu.bandcamp.com/track/oenera#b58688871",
        "@type": [
          "MusicRelease",
          "Product"
        ],
        "additionalProperty": [
          {
            "@type": "PropertyValue",
            "name": "item_id",
            "value": 58688871
          },
          {
            "@type": "PropertyValue",
            "name": "item_type",
            "value": "b"
          },
          {
            "@type": "PropertyValue",
            "name": "selling_band_id",
            "value": 1309392494
          },
          {
            "@type": "PropertyValue",
            "name": "type_name",
            "value": "Digital"
          },
          {
            "@type": "PropertyValue",
            "name": "art_id",
            "value": 1922551329
          },
          {
            "@type": "PropertyValue",
            "name": "is_bfd",
            "value": true
          }
        ],
        "image": [
          "https://f4.bcbits.com/img/a1922551329_10.jpg"
        ],
        "musicReleaseFormat": "DigitalFormat",
        "name": "full digital discography (41 releases)",
        "offers": {
          "@type": "Offer",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "bundle_size",
              "value": 41
            },
            {
              "@type": "PropertyValue",
              "name": "discount",
              "value": 0.35
            }
          ],
          "availability": "OnlineOnly",
          "price": 54.64,
          "priceCurrency": "EUR",
          "priceSpecification": {
            "minPrice": 54.64
          },
          "url": "https://gutkeinforu.bandcamp.com/track/oenera#b58688871-buy"
        }
      }
    ],
    "albumReleaseType": "SingleRelease",
    "name": "OENERA",
    "numTracks": 1
  },
  "keywords": [
    "Electronic",
    "keta techno",
    "meta",
    "techno",
    "trance"
  ],
  "mainEntityOfPage": "https://gutkeinforu.bandcamp.com/track/oenera",
  "name": "OENERA",
  "publisher": {
    "@id": "https://gutkeinforu.bandcamp.com",
    "@type": "MusicGroup",
    "additionalProperty": [
      {
        "@type": "PropertyValue",
        "name": "band_id",
        "value": 1309392494
      },
      {
        "@type": "PropertyValue",
        "name": "has_any_downloads",
        "value": true
      },
      {
        "@type": "PropertyValue",
        "name": "has_download_codes",
        "value": true
      },
      {
        "@type": "PropertyValue",
        "name": "image_height",
        "value": 1364
      },
      {
        "@type": "PropertyValue",
        "name": "image_id",
        "value": 33148176
      },
      {
        "@type": "PropertyValue",
        "name": "image_width",
        "value": 1365
      }
    ],
    "description": "support a tiny acquarium on the tip of a K-hall ~)",
    "genre": "https://bandcamp.com/discover/electronic",
    "image": "https://f4.bcbits.com/img/0033148176_10.jpg",
    "mainEntityOfPage": [
      {
        "@type": "WebPage",
        "name": "SoundCloud",
        "url": "https://soundcloud.com/gutkein"
      },
      {
        "@type": "WebPage",
        "name": "Instagram",
        "url": "https://www.instagram.com/di_gutkein/"
      }
    ],
    "name": "GUTKEIN",
    "subjectOf": [
      {
        "@type": "WebPage",
        "additionalProperty": [
          {
            "@type": "PropertyValue",
            "name": "nav_type",
            "value": "m"
          }
        ],
        "name": "music",
        "url": "https://gutkeinforu.bandcamp.com/music"
      },
      {
        "@type": "WebPage",
        "additionalProperty": [
          {
            "@type": "PropertyValue",
            "name": "nav_type",
            "value": "c"
          }
        ],
        "name": "community",
        "url": "https://gutkeinforu.bandcamp.com/community"
      }
    ]
  }
}
