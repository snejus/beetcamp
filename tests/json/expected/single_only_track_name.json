{
  "albumtype": "single",
  "albumtypes": "single",
  "arranger": null,
  "artist": "GUTKEIN",
  "artist_credit": null,
  "artist_id": "https://gutkeinforu.bandcamp.com",
  "artist_sort": null,
  "bpm": null,
  "catalognum": "",
  "comments": "little distorted",
  "composer": null,
  "composer_sort": null,
  "country": "XW",
  "data_source": "bandcamp",
  "data_url": "https://gutkeinforu.bandcamp.com/track/oenera",
  "day": 10,
  "disctitle": null,
  "genre": "techno, trance",
  "index": null,
  "initial_key": null,
  "label": "GUTKEIN",
  "length": 355,
  "lyricist": null,
  "mb_workid": null,
  "media": "Digital Media",
  "medium": null,
  "medium_index": null,
  "medium_total": null,
  "month": 1,
  "release_track_id": null,
  "style": "electronic",
  "title": "OENERA",
  "track_alt": null,
  "track_id": "https://gutkeinforu.bandcamp.com/track/oenera",
  "work": null,
  "work_disambig": null,
  "year": 2021
}
